api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -796,6 +796,130 @@
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-5">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="creator"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="creator" type="int">
+				<constraints nullable="false"/>
+			</column>
+		</addColumn>
+		<addForeignKeyConstraint baseColumnNames="creator" baseTableName="cohort_member" constraintName="cohort_member_creator" deferrable="false" initiallyDeferred="false" referencedColumnNames="user_id" referencedTableName="users"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-6">
+		<preConditions>
+			<not>
+				<primaryKeyExists tableName="cohort_member"/>
+			</not>
+		</preConditions>
+		<addPrimaryKey tableName="cohort_member" columnNames="cohort_member_id"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171246-7">
+		<addAutoIncrement tableName="cohort_member" columnName="cohort_member_id" columnDataType="int"/>
+	</changeSet>
+	<changeSet id="201610042145-1" author="vshankar">
+        <preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="date_created"/>
+			</not>
+		</preConditions>
+        <addColumn tableName="cohort_member">
+			<column name="date_created" type="DATETIME">
+				<constraints nullable="false"/>
+			</column>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-1">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="voided"/>
+			</not>
+		</preConditions>
+        <addColumn tableName="cohort_member">
+			<column name="voided" type="BOOLEAN">
+				<constraints nullable="false"/>
+			</column>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-2">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="voided_by"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="voided_by" type="int"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-3">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="date_voided"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="date_voided" type="DATETIME"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-4">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="void_reason"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="void_reason" type="varchar(255)"/>
+		</addColumn>
+	</changeSet>
+	<changeSet id="201610042145-2" author="vshankar">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="uuid"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="uuid" type="char(38)">
+				<constraints nullable="false"/>
```

Comment:
Same here, you might to look in this file for functions to call to generate uuids